include/mitsuba/core/transform.h

```diff
@@ -104,90 +133,6 @@ template <typename Point_> struct Transform {
         return dr::all_nested(matrix != t.matrix);
     }
 
-    /**
-     * \brief Transform a 3D vector/point/normal/ray by a transformation that
-     * is known to be an affine 3D transformation (i.e. no perspective)
-     */
-    template <typename T>
-    MI_INLINE auto transform_affine(const T &input) const {
```

Comment:
Should we maybe keep this with a deprecation warning? (like in Python)

Reply:
Fair enough.